src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs

```diff
@@ -41,15 +41,25 @@ public async Task<Result<InvitationDto>> Handle(GetInvitationByIdQuery request,
 
             if (createdBy == null)
             {
-                throw new Exception($"Could not get person that created the IPO with id {invitation.CreatedById}");
+                return new NotFoundResult<InvitationDto>($"Could not get person that created the IPO with id {invitation.CreatedById}");
             }
 
             var createdByName = createdBy.FirstName + ' ' + createdBy.LastName;
 
-            var meeting = await _meetingClient.GetMeetingAsync(invitation.MeetingId, query => query.ExpandInviteBodyHtml().ExpandProperty("participants.outlookstatus"));
-            if (meeting == null)
+            GeneralMeeting meeting;
+            try
             {
-                throw new Exception($"Could not get meeting with id {invitation.MeetingId} from Fusion");
+                meeting = await _meetingClient.GetMeetingAsync(invitation.MeetingId,
+                    query => query.ExpandInviteBodyHtml().ExpandProperty("participants.outlookstatus"));
+
+                if (meeting == null)
+                {
+                    return new NotFoundResult<InvitationDto>($"Could not get meeting with id {invitation.MeetingId} from Fusion");
```

Comment:
If there's an exception, we continue, but with an empty meeting we return? Is that correct?

Reply:
This is correct (from the information I have now). The reason:
If meeting == null, this means that the meeting does not exist in fusion at all, and something has gone wrong :(

If we get an exception, then it means that we got some other error, most likely the person is not directly invited to the meeting.

Do you think we should keep getting IPO even if meeting is null, or keep it like it is?

Reply:
I will change it so that we are not dependant on Fusion!